aten/src/ATen/native/quantized/cpu/qembeddingbag.cpp

```diff
@@ -108,33 +108,61 @@ at::Tensor& embedding_lookup_fallback_impl(
         const uint8_t* scale_bias =
             weight_data + (idx + 1) * weight_size - 2 * sizeof(float);
         uint32_t scale_val_int32 = 0;
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Comment:
Hmm, if `__BYTE_ORDER__` is not define for some reason, it will go to big-ending case, wouldn't it? I.e. can we switch the default to assume LE